src/openforms/js/components/admin/form_design/logic/constants.js

```diff
@@ -0,0 +1,76 @@
+// TODO: these are the built in json logic operators, but it's possible to define
+// custom operators. See https://jsonlogic.com/operations.html
+// We start with a minimal supported set based on the user stories (and common sense).
+// The data structure is a mapping of the actual json logic operator as key and the
+// human readable label as value.
+const OPERATORS = {
+    '==': 'is equal to',
+    '!=': 'is not equal to',
+    '>': 'is greater than',
+    '>=': 'is greater than or equal to',
+    '<': 'is less than',
+    '<=': 'is less than or equal to',
+    'in': 'in', // array or string (!)
+};
+
+
+// map the Formio component types to allowed operators. Strings cannot be compared
+// against each other for >=,>,<,<= etc., so this includes a logical allow-list to only
+// show relevant components.
+const COMPONENT_TYPE_TO_OPERATORS = {
+    number: [
+        '==',
+        '!=',
+        '>',
+        '>=',
+        '<',
+        '<=',
+    ],
+    textfield: [
+        '==',
+        '!=',
+        'in',
+    ],
+    iban: [
+        '==',
+        '!=',
+        'in',
+    ],
+};
+
+const ACTION_TYPES = [
+    ['disable-next', 'disable continuing to the next form step.'],
+    ['property', 'change a property of a component.'],
+    ['value', 'change the value of a component']
+];
+
+
+// Action types that once they are selected need further configurations.
+// For example, picking which property of a component should be changed.
+const ACTIONS_WITH_OPTIONS = [
+    'property',
+    'value'
+];
+
+
+const MODIFIABLE_PROPERTIES = [
+    ['required', 'required'],
+    ['hidden', 'hidden'],
+    ['disabled', 'disabled']
+];
+
+
+const PROPERTY_VALUES = [
+  ['true', 'Yes'],
+  ['false', 'No']
```

Comment:
Can we use actual booleans here so that they're properly serialized in JSON too?